corehq/ex-submodules/casexml/apps/stock/consumption.py

```diff
@@ -89,8 +91,11 @@ def compute_daily_consumption(case_id,
         window_start,
         window_end
     )
+    sql_product = SQLProduct.objects.get(product_id=product_id)
+    domain = sql_product.domain
```

Comment:
is this just to get access to the domain for the toggle? perhaps you could put it in the `ConsumptionConfiguration` object instead?